src/datepicker/examples/bootstrap.datepicker.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @ngdoc directive
+ * @name patternfly.datepicker.componenet:pfBootstrapDatepicker
+ * @element pf-bootstrap-datepicker
+ *
+ * @param {date} date must be a Javascript Date - to be displayed in the input. Can be left empty.
+ * @param {string} format default date format for displayed dates
+ * @param {object} dateOptions an object to configure the uib-datepicker in one place (optional)
+ * @param {boolean} isOpen whether or not to show the datepicker (default false) (optional)
+ * @param {object} popupPlacement Placement of the datepicker popup relative to the input element. Passing in 'auto' separated by a space bfore the placement will enable auto positioning. (optional)
+ *
+ * @description
+ * A wrapper for the Angular UI {@link http://angular-ui.github.io/bootstrap/#!#datepickerPopup datepicker}.
```

Comment:
That link isn't  working for me, it goes to the top of Angular UI Bootstrap page.   I think `http://angular-ui.github.io/bootstrap/#datepickerPopup` will be enough to link-n-launch to the correct part of the page.
Hmm..still not working in some cases.  Funny, even their [GitHub](https://github.com/angular-ui/bootstrap/tree/master/src/datepickerPopup) page's 'here' and 'inline datepicker' links do not work correctly for me.

Reply:
yeah - i've noticed the same problem as I've tried linking to the param documentation in response to Joe's comments... Not sure what to do about that.